recipes/xbpch/meta.yaml

```diff
@@ -0,0 +1,54 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "xbpch" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "903358ec30cd6ca8c577aa0c9f2b384574f95c525452911b53614c5ae2046271" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - dask
+    - xarray
+
+test:
+  imports:
+    - xbpch
+
+about:
+  home: http://github.com/darothen/xbpch
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
Is the `LICENSE` file in the tarball? It does not look like it.
If not this won't work.

Reply:
The error now is b/c conda cannot find the `LICENSE` you specified.
I checked and it is not in the tarball. You can comment this line and add the `LICENSE`
to your `MANIFEST.in` in a future release.